src/api/libcellml/variable.h

```diff
@@ -55,8 +55,20 @@ class LIBCELLML_EXPORT Variable: public NamedEntity
     UnitsPtr getUnits() const;
 
     /**
+     * @brief Set the initial value for this variable using a string.
+     * Set the initial value for this variable using a string.
+     *
+     * @overload
+     * @sa getInitialValue
+     * @param initialValue the initial value to set.
```

Comment:
"The initial..." rather than "the initial..."

Note: there are other similar instances here and there throughout the whole code.